wapitiCore/definitions/dangerous_resource.py

```diff
@@ -24,6 +24,8 @@
 NAME = _("Potentially dangerous file")
 SHORT_NAME = NAME
 
+WSTG_CODE = ["WSTG-CONF-04"]
+
```

Comment:
Related to the use of vulnerable compoents : WSTG-CONF-01
https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/01-Test_Network_Infrastructure_Configuration